app/helpers/statistics_helper.rb

```diff
@@ -4,13 +4,21 @@
 
 module StatisticsHelper
   def registrations_status statistics
-    if statistics.open_registrations?
+    if !!statistics.open_registrations?
```

Comment:
* If you can call `open_registrations?` on `statistics` that's certainly not a symbol.
* If  `open_registrations?` returns a symbol, `!!symbol` is always `true`.
* if `open_registrations?` returns nil or a symbol, nil is falsey and the symbol is truthy, so the `!!` is unneeded.